src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -95,7 +95,8 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                     items.AddRange(GetListHeaderItems(span));
                 }
 
-                if (token.Parent.Parent is DocumentationCommentTriviaSyntax)
+                if (token.Parent.Parent is DocumentationCommentTriviaSyntax ||
+                    (token.Parent.Parent.IsKind(SyntaxKind.XmlEmptyElement) && token.Parent.Parent.Parent is DocumentationCommentTriviaSyntax))
```

Comment:
I saw that actually, and just figured it was some voodoo that someone was doing so I left it alone. 😉